apis/MessagesEndpoint.es6.js

```diff
@@ -0,0 +1,48 @@
+import Comment from '../models2/CommentModel';
+import Post from '../models2/PostModel';
+import Message from '../models2/MessageModel';
+import { runQuery, save } from '../apiBase/APIRequestUtils';
+
+const TYPE_MAP = {
+  't1': Comment,
```

Comment:
🔕  I wonder more and more if we should use the constants versions of these. Outside the scope of this though.